packages/config/src/configs/peaqEvmAlphanet.ts

```diff
@@ -0,0 +1,41 @@
+import { BalanceBuilder } from '@moonbeam-network/xcm-builder';
+import { dev, ftmwh } from '../assets';
+import { moonbaseAlpha, peaqEvmAlphanet } from '../chains';
+import { AssetConfig } from '../types/AssetConfig';
+import { ChainConfig } from '../types/ChainConfig';
+
+export const peaqEvmAlphanetConfig = new ChainConfig({
+  assets: [
+    new AssetConfig({
+      asset: ftmwh,
+      balance: BalanceBuilder().evm().erc20(),
+      // contract: ContractBuilder().Xtokens().transfer(),
```

Comment:
left overs?

Reply:
This doesn't matter for now and will be fixed later.